### DISCUSSION TITLE
Nextjs not creating font variable on production environment

### DISCUSSION THREAD

**Discussion by sudhinjyothis** [vercel/community#5900] [Help]
### Summary

When I use the next/font in my project, the variable is getting declared while running locally but not generating on production  

When running in localhost
![Screenshot 2024-02-22 161131](https://github.com/vercel/community/assets/64127862/995acee5-84be-40ae-b5c7-54433174decb)

in production
![Screenshot 2024-02-22 161204](https://github.com/vercel/community/assets/64127862/7bdf0e05-aa71-4159-80f7-cf35714859fd)

### Steps to Reproduce

just used the font component

**Comment by amyegan** [comment]
Hey @sudhinjyothis. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that you can share?

**Reply by sudhinjyothis** [reply]
@amyegan  when i redeployed the same version things got sorted out thank you
